src/main/webapp/xml/xml2json.xslt

```diff
@@ -50,14 +50,20 @@
          <xsl:text>"component" : "</xsl:text>
          <xsl:value-of select="component" />
          <xsl:text>",</xsl:text>
-         <xsl:text>"labels" : "</xsl:text>
+         <xsl:text>"labelsFlat" : "</xsl:text>
```

Comment:
It's been years since I looked at XSL. Am I correct in reading this as you've added a non-parsed string version of the labels to the output in addition to an array version?  Both of those replacing the original space-delimited string version.